modeling/dataset/samplers/base.py

```diff
@@ -104,16 +106,13 @@ def __getitem__(self, index):
 
 class MultiDomainEvalSampler(EvalSampler):
 
-    def __init__(self, dataset, num_users, num_items, target_domain):
+    def __init__(self, dataset, num_users, num_items):
```

Comment:
I think you should still pass `target_domain` and `other_domains` here for its initialization.